lib/config/options/index.ts

```diff
@@ -262,7 +262,7 @@ const options: RenovateOptions[] = [
   {
     name: 'binarySource',
     description:
-      'Controls whether third party tools like npm or Gradle are called directly, or via Docker sidecar containers.',
+      'Controls whether third-party tools like npm or Gradle are called directly, or via Docker sidecar containers.',
```

Comment:
I ran `yarn jest -u lib/config` on Gitpod which passes, and did not change any spec files. I'm marking this PR as draft until I know that the full test suite works okay. 😄 

Do I need to change the PR title from `docs:` to `fix:` because I'm changing a line in the code?

Reply:
No, it's fine. Docs deploy after every commit anyway